decentraland/unity-explorer
# Fix: terrain details color blend

## What does this PR change?

fix #736
fix #787

- reset Wind prefab to zeros, so it is not far away
- fix issue when jumping to second/third world was without any terrain

...

## How to test the changes?

1. Launch the explorer
2. Check grass is not red (but orange that blends into terrain), for example at -12,7
3. Goto some world -
4. go back to genesis 
5. go to some world -> verify  terrain is presented
6. verify that Trees are effected by the wind


## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
